app/src/main/java/com/my/journeyplanner/helpers/ResponseCodeInterceptor.kt

```diff
@@ -0,0 +1,14 @@
+package com.my.journeyplanner.helpers
+
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class ResponseCodeInterceptor : Interceptor {
```

Comment:
Better call it something we know what it is. Say like, MultiOptionResponseCodeInterceptor or something

Reply:
Updated as advised